.gitignore

```diff
@@ -12,7 +12,8 @@
 /spec/examples.txt
 /test/tmp/
 /test/version_tmp/
-/tmp/
+/tmp/*
+!tmp/.gitkeep 
```

Comment:
Why do we need this file? Couldn't we create/destroy this directory as needed locally?

Reply:
Good point. I don't think this is necessary since the `visit_rendered_component_in_browser` method is now using `Tempfile` which would erase the temporary file.